### DISCUSSION TITLE
Does not work backup and restore in Teampass 3.1.2

### DISCUSSION THREAD

**Discussion by margeu** [nilsteampassnet/TeamPass#4089] [General]
After Backup, if I try to Restore getting the following error in red alert:

 **Error caught
An error appears. Answer from Server cannot be parsed!
Returned data:
true** 

![backup error](https://github.com/nilsteampassnet/TeamPass/assets/32653608/a7126f16-3454-403f-9ff8-8b3e47ce2d23)

The only error I detect connected to the restore function is in the log /var/log/php-fpm/www-error.log with the following message

**[06-Mar-2024 16:59:43 UTC] /tmp/phpGng0R2
[06-Mar-2024 16:59:46 UTC] PHP Warning:  Undefined array key "post_totalSize" in /var/www/html/TeamPass/sources/backups.queries.php on line 292**

The configuration is:
Web server: httpd-2.4.57
Database: mariadb 10.5.22
PHP version: PHP 8.1
Operating system: RHEL 9.3

Has anyone successfully tried backup and restore with version 3.1.2 ?

Thank you for your feedback

**Comment by lopo215** [comment]
Same problem

**Comment by e4rthdog** [comment]
Same here.

**Comment by ivancasanovaicp** [comment]
Same

**Comment by vura** [comment]
Same here.

Any solution?

@nilsteampassnet

**Comment by vura** [comment]
Up

**Comment by gexpo** [comment]
is this issue still open?? 
i'm experiencing the same problem after upgraded from 3.1.4.29 to 3.1.4.30 - very strange -
